packages/server/graphql/queries/suggestedIntegrations.ts

```diff
@@ -33,33 +38,30 @@ export default {
         return standardError(new Error('Not on same team as user'), {userId: viewerId})
       }
     }
-    const teamIntegrationsByUserId = await getTeamIntegrationsByUserId(teamId)
+    const permLookup: PermLookup = await getPermsByTaskService(dataLoader, teamId, userId)
```

Comment:
-1 let's remove the `PermLookup` here as the type is already inferred automatically